syntax = "proto3";

package llm;

service LLMService {
  rpc getLLMAnswer(LLMRequest) returns (LLMResponse);
}

message LLMRequest {
  string requestId = 1;
  string query = 2;
  string context = 3;
}

message LLMResponse {
  string requestId = 1;
  string answer = 2;
}
